components/basecomponents/SEO.tsx

```diff
@@ -3,25 +3,26 @@ import Head from 'next/head';
 
 interface Props {
   title: string;
-  description: string;
-  image: string;
+  description?: string;
+  image?: string;
+  noreferrer?: boolean;
 }
 
-const SEO = ({ title, description, image }: Props) => {
+const SEO = ({ title, description, image = 'https://www.movapp.cz/icons/movapp-cover.jpg', noreferrer = true }: Props) => {
   return (
     <Head>
-      <meta name="referrer" content="no-referrer" />
+      {noreferrer && <meta name="referrer" content="no-referrer" />}
       <title>{title}</title>
-      <meta name="description" content={description} />
+      <meta name="description" content={description ? description : title} />
```

Comment:
Try this instead: `description ?? title` :) 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Also, just assign it to a variable and re-use it: 

`const appliedDescription = description ?? title;`

and then 

` <meta name="og:description" content={appliedDescription} />`